### ISSUE TITLE
Code Table Request - REMOVE tested for presence

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6228]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

[tested for presence](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#tested_for_presence) might be another way of saying https://arctos.database.museum/info/ctDocumentation.cfm?table=ctexamined_detected ??

Data:
[temp_tfp.csv.zip](https://github.com/ArctosDB/arctos/files/11384080/temp_tfp.csv.zip)

Summary:

```

 guid_prefix | count 
-------------+-------
 UAM:Env     |   334

```

Other Summary:

```

arctosprod@arctos>> select attribute_value,count(*) from temp_tfp group by attribute_value;
 attribute_value | count 
-----------------+-------
 absent          |   274
 present         |    60
(2 rows)

```

Users:

 @kyndallh

_**Context**: Describe why this new value is necessary and existing values are not._

We might have two ways of doing the same thing.

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

Code table cleanup should always be highest priority.

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality)._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by campmlc** [comment]
We need a value to convert these to in the Examined_Detected code table. 
Replacement attribute would be 
Examined for: eDNA
Detected: eDNA
Not Detected: eDNA
@KyndallH

**Comment by Jegelewicz** [comment]
Blocked by #6263

**Comment by Jegelewicz** [comment]
That term has to be added before this can proceed.

**Comment by dustymc** [comment]
Fresh data:


[temp_tfp.csv](https://github.com/user-attachments/files/28977955/temp_tfp.csv)

**Comment by mkoo** [comment]
Reviving this conversation: agreed that this term needs to be removed. The question is how to accommodate its values in the Examinedfor table.

Suggestions of eDNA appear to me at first blush not exactly right-- maybe examined for = vertebrate (and add in remarks to specify)

That would have a path forward for UAM:Env

However now that this issue is in kindergarten, we will talk to Cam about the UAMb:Herb, UAM:Herb datasets (above )
@camwebb email me a time to talk?

**Comment by campmlc** [comment]
I agree with the original request except that we do need the eDNA value added to the examined detected code table as requested above. Genomic collections must have a way to record that an environmental sample cataloged as soil, water or air was tested for eDNA or environmental DNA. This is specifically an examination for the presence of environmental DNA. Later this DNA, if present, may be screened for the presence of pathogens or other species, which can be addressed through other means. But adding eDNA to the code table is a simple solution that allows collections managing these samples to move forward now using a standard consistent with global discussions across the broader genomics community.

**Comment by campmlc** [comment]
One.of the functional requirements of this attribute and code table is to allow recording of what is known at the time of cataloging. At data entry, it may not be known what the results of the examination are in detail, only that it was examined for eDNA and that eDNA was detected or not. More detailed information on what species were or were not detected can added later through attributes, identifiers, and relationships.

**Comment by camwebb** [comment]
(@mkoo I have emailed.) We've been using 'tested for presence' to indicate the specimen (or one of its parts) has been sampled for DNA. When I started, I could not find a more appropriate attribute. Is there now one? It seems like such a common need (i.e., to mark a record as having been sampled for DNA).

**Comment by campmlc** [comment]
@camwebb We subsample parts for DNA extraction all the time, but we record these through loans. A loan request leads to a part subsample which then has a further subsample of DNA. That way we track the parent/child relationships, and the usage via the loan, which can accommodate all the metadata of the sampling protocol, agents, dates etc.

**Comment by camwebb** [comment]
@campmlc We do this loan/subsample method too, but often we just find a "DNA sampled..." annotation label on a specimen which is not associated with any known loan in Arctos. To then create a dummy loan, subsample a specimens, etc. is very time-consuming, and beyond what we can expect of ourselves, and muddies the quality of real loans. We need a simple way to flag the record as having an annotation label saying it was sampled by X for Y DNA. An attribute seems like the best approach. Maybe we should just use the default, general 'verbatim attribute' for this?  But tested for presence seemed like a better choice.

**Comment by dustymc** [comment]
@camwebb maybe a new state for 

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctprocessing_history

makes sense.  I can unload to the attribute bulkloader format, or just update if there's an appropriate target - want to start that Issue?

@mkoo will you file an issue for "vertebrate" or ??

**Comment by camwebb** [comment]
> ctprocessing_history

Processing history doesn't seem right to me - that feels more appropriate for recording the history of the specimen as a museum object, rather than a once-alive biological entity.  What I'd really like is a whole new attribute, 'annotations' with controlled vocab: 'identification', 'DNA sampled',... to allow modeling of any bit of paper stuck onto a herbarium sheet. (Note, this is a different use of "annotations" from [this](https://handbook.arctosdb.org/how_to/annotations.html).) I don't think other disciplines have the same physical annotations on their specimens, and that may explain why 'annotations' does not yet exist. I have played with modeling these annotations as `specimen parts` of type `media`, but that's clumsy and an annotation is not really _part_ of a collection object.

Do you (@dustymc, @mkoo) think this ('annotations' attribute) might be worth a new CT discussion?

**Comment by dustymc** [comment]
>specimen parts of type media

FWIW: parts are "things to which one might stick barcodes" so if you have a USB drive or glass slide or chunk of paper then that's probably "correct." (Meaning "using the model as designed" not necessarily "useful" - and I'm officially ignoring ``media`` being used for digital stuff and who knows what else, and hopefully that use won't find some way to bite anyone.)

>other disciplines have the same physical annotations on their specimens

I think yes/no: lots of notes in old boxes and scrawlings on bones and etc., but I'm not aware of anything to the same extent as what happens when there's a nice big flat thing to write on and glue stuff to available.


>Processing history ... feels

Ugh, that attribute is currently defined as....

>processing history

... so yea. Can we re-shape it into something that works here? And I'll open a new Issue.

>worth a new CT discussion

Sure, please start with the usual functional needs. 

(Or maybe these have been "tested for annotations"???? There's definitely some cost to more THINGS so reuse is nice when possible, but there are also discoverability etc. costs to hammering square pegs into round holes, and IDK where this might fall on that spectrum.)

**Comment by mkoo** [comment]
please see intake form for updates and action items

**Comment by camwebb** [comment]
After last week's call with @mkoo, I approved converting all `tested for presence` attributes to `examined for` -> `DNA`. Good solution.

But... it looks like the `tested for presence` attribute has _already_ gone, along with all our data! (From memory, this attribute was set for ~350 records). Searching for `examined for` = `DNA` brings up nothing.  I only just now activated `examined for` as allowed for UAM:Herb and UAMb:Herb.  HELP. I hope this attribute data is recoverable.

**Comment by dustymc** [comment]
@camwebb check the linked roadmap issue, the data are stashed there and @mkoo has a spreadsheet to reload them cooking. 

FYI anything with this milestone should have a linked road map issue, this one is up yonder....

<img width="753" height="144" alt="Image" src="https://github.com/user-attachments/assets/28442460-c451-49ab-a3ab-297bbc1b9fd3" />

or https://github.com/ArctosDB/arctos-roadmap/issues/22

**Comment by mkoo** [comment]
apologies Cam, slow to load the new attributes -- pop to top of list now!
no data was lost!

**Comment by mkoo** [comment]
@camwebb there are two records that need your attention, then migration complete:

UAM:Herb:12326 : 
Tissue removed for SEM study by Emily Molhoek and Sarah Fowell; UAF - Geol. and Geophysics Dept. March 2004 (might have to propose new examined for: morphology ?)

UAM:Herb:245168: 
Leaf sampled for cytological analysis by Naoki Takebayashi, March 2025 (maybe examined for: pathology)

See the rest in the [prep sheet (tab called problems)](https://docs.google.com/spreadsheets/d/1VDJSb9jzv-z2gl3yHiwVBWVs7U6MQsltmdCh6iLPoqg/edit?gid=600370841#gid=600370841)

**Comment by mkoo** [comment]
major work completed.
